util/topgen/intermodule.py

```diff
@@ -963,16 +964,28 @@ def im_netname(sig: OrderedDict,
 
     # External signal handling
     if "external" in obj and obj["external"]:
-        pairs = {
-            # act , suffix: additional suffix
-            ("req", "req"): "_o",
-            ("req", "rsp"): "_i",
-            ("rsp", "req"): "_i",
-            ("rsp", "rsp"): "_o",
-            ("req", ""): "_o",
-            ("rcv", ""): "_i",
-            ("none", "io"): "_io"
-        }
+        if obj["conn_type"]:
```

Comment:
i honestly can't remember what was done here,
But if it avoids a multi-driven signal who am i to complain :)

thanks for the fix! did this come up as an issue because suspend goes to both another module and top level?

Reply:
Yup, that's exactly the spot!

It was the only signal of its type in `top_earlgrey`, so I guess we just never encountered this situation before. I happened to spot that we'd already detected the condition and saved this 'conn_type' attribute into `top['inter_signal']['external']`, but it didn't get into the `inter_signal_list` for modules.

Reply:
sounds good!